README.md

```diff
@@ -224,7 +224,30 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 | v0.29.0                | v1.19 to v1.21       |
 | v0.28.0                | v1.19 to v1.21       |
 | v0.27.0                | v1.19 to v1.21       |
-| v0.26.0                | v1.18 to v1.20       |
+
+
+### OpenTelemetry Operator vs. Cert Manager
+Since Cert-Manager deprecated *cert-manager.io/v1alpha2, cert-manager.io/v1alpha3, cert-manager.io/v1beta1, acme.cert-manager.io/v1alpha2, acme.cert-manager.io/v1alpha3, acme.cert-manager.io/v1beta1* APIs on version 1.4.0, we decided to create a compability Matrix between OpenTelemetry Operator and Cert-manager.
+
+| OpenTelemetry Operator | Cert-Manager         |
+|------------------------|----------------------|
+| v0.37.1                | v1.4.0 to v1.6.1     |
```

Comment:
Interesting, I thought 1.6.x would only work with 0.38.0 and on. Did you test this combination?

Reply:
The [479](https://github.com/open-telemetry/opentelemetry-operator/pull/479) has updated the cert-manager files related, which could have been done since `cert-manager: v.1.4.0`.